CyberSource/cybersource-rest-client-python
# Fixes Issue #20: Remove Typing from requirements for Python 3.7

## Issue

[Issue #20: Exception: Callable has no attribute __abc_registry](https://github.com/CyberSource/cybersource-rest-client-python/issues/20)

## What

Users receive a `Callable has no attribute __abc_registry` error.

This is a [known issue](https://github.com/python/typing/issues/573) when installing typing on a system that has Python >= 3.7. It will result in an error.

## Test

1. Installed the latest available version of the SDK through pip so it will install all required dependencies too.
2. Create a test.py file and import the Cybersource SDK in an environment with Python >= 3.7.
3. Execute the script python or upload the code to Lambda and run it. It should no longer fail.
